bagitobjecttransfer/recordtransfer/enums.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
+
+
+class TransferStep(Enum):
+    """Steps in the multi-stage transfer form, listed in the order they should be completed.
+    Note that the final step is either "uploadfiles" or "finalnotes" depending on the
+    configuration.
+    """
+
+    ACCEPT_LEGAL = "acceptlegal"
+    CONTACT_INFO = "contactinfo"
+    SOURCE_INFO = "sourceinfo"
+    RECORD_DESCRIPTION = "recorddescription"
+    RIGHTS = "rights"
+    OTHER_IDENTIFIERS = "otheridentifiers"
+    GROUP_TRANSFER = "grouptransfer"
+    UPLOAD_FILES = "uploadfiles"
+    FINAL_NOTES = "finalnotes"
+
+    @classmethod
+    def get_formset_steps(cls) -> list["TransferStep"]:
+        """Return the steps of the multi-stage form which use formsets."""
+        return [TransferStep.RIGHTS, TransferStep.OTHER_IDENTIFIERS]
```

Comment:
I recommend removing this function (see my later comments for why)

Reply:
Resolved by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/243/commits/917757cfee3f8423c9d3047f56c81233dfbab1c3